ckanext/ontario_theme/templates/internal/header.html

```diff
@@ -204,7 +205,7 @@
                 <button class="ontario-hide-for-large ontario-header__menu-toggler ontario-header-button ontario-header-button--with-outline {{ 'fr' if current_lang == 'fr' }}"
                         id="ontario-header-menu-toggler"
```

Comment:
There are 2 buttons active at the same time with the same id's:
- id="ontario-header-menu-toggler"
- id="ontario-header-nav-toggler"

<img width="730" alt="Screenshot 2023-10-06 at 09 58 08" src="https://github.com/ongov/ckanext-ontario_theme/assets/1254764/f7a2380e-cfe7-4309-a062-b8cd080b9151">
<img width="455" alt="Screenshot 2023-10-06 at 09 58 30" src="https://github.com/ongov/ckanext-ontario_theme/assets/1254764/7f31d7cf-93a8-4f50-bb7d-a35723b45c71">

when on mobile screen for the same "X" button:
<img width="372" alt="Screenshot 2023-10-06 at 09 59 20" src="https://github.com/ongov/ckanext-ontario_theme/assets/1254764/7ea7277a-4fcc-4539-b323-4bbcafec8ce4">

Reply:
This is consistent with the design system [application header](https://designsystem.ontario.ca/components/preview/application-header.html)

Reply:
huh